request_data.py

```diff
@@ -0,0 +1,54 @@
+import emission.core.get_database as edb
+import bson.json_util as bju
+import requests
+import json
+import argparse
+import logging
+
+parser = argparse.ArgumentParser()
+parser.add_argument("from_date",
+        help="from_date in the format of YYYY-MM-DD")
```

Comment:
the `metadata.write_ts` for this entry is `2016-06-30T07:00:19` in UTC. So the input dates should be local time. I will make the clarification in the new script.